docs/release-engineering.md

```diff
@@ -12,10 +12,23 @@ CircleCI will do most of the work for you. You will need to edit the draft relea
 Please change the version number as appropriate.  Substitute (for example)
 `3.20.0` any place you see `VERSION` in this doc.
 
-## Step 1. Tag the commit in master that you want to release
+## Step 1. Rebuild generated files
 
 ```shell
 git checkout master
+go generate
```

Comment:
I make a note to myself to do one last `git pull`. Maybe you also think this is a good idea and can you include it as a necessary step?

```suggestion
git pull
go generate
```